pkg/cmd/openshift-tests/collect-disk-certificates/collector.go

```diff
@@ -161,27 +157,16 @@ func (o *RunCollectDiskCertificatesOptions) Run(ctx context.Context) error {
 	pkiList := &certgraphapi.PKIList{}
 	errs := []error{}
 
-	controlPlaneLabel := labels.SelectorFromSet(map[string]string{"node-role.kubernetes.io/control-plane": ""})
-
-	nodeList, err := o.KubeClient.CoreV1().Nodes().List(ctx, metav1.ListOptions{LabelSelector: controlPlaneLabel.String()})
-	if err != nil {
-		return fmt.Errorf("failed to list nodes: %v", err)
-	}
-	masters := []*corev1.Node{}
-	for i := range nodeList.Items {
-		masters = append(masters, &nodeList.Items[i])
-	}
-
 	for _, srcDir := range o.CollectDirs {
-		dirPKIList, err := certgraphanalysis.GatherCertsFromDisk(ctx, o.KubeClient, o.Prefix, srcDir,
+		dirPKIList, err := certgraphanalysis.GatherCertsFromDisk(ctx, o.KubeClient, srcDir,
 			certgraphanalysis.ElideProxyCADetails,
 			certgraphanalysis.SkipRevisioned,
 			certgraphanalysis.SkipHashed,
-			certgraphanalysis.RewriteNodeIPs(masters))
```

Comment:
expected to continue rewriting node ips

Reply:
They would be rewritten in the final report - but not in the report sent from the node. This would allow us to avoid requiring node access from the pod and be able to process it once at deduplication stage